src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -154,6 +149,66 @@ docker-compose --verbose up --exit-code-from helloworld
 key *docker-compose-options-split-regex*. If you supply e.g. "!SPLIT!" as value, then your *docker-compose-up-options* will need to look like this: "--option1!SPLIT!--option2".
 
+=== Docker_File task
+
+The basic behavior of a Docker_File task is to create an image with Docker build and to create a container based on the built image. At the end of the job, the created image and container are deleted. Advanced options will allow to parametrize these actions.
+
+In order for Docker_File tasks to work, the _Node_ must have Docker installed. Please refer to the official
+ Docker documentation to see how to install http://docs.docker.com/engine/installation/[Docker^]
+
+A Docker_File task expects the content of a Dockerfile inside the _Script_ section. You can find out how to write Dockerfile
+in the official https://docs.docker.com/engine/reference/builder/[Dockerfile documentation^].
+
+
+The Docker_File task will run a combination of docker commands to build, start, stop and remove containers. 
+
+To get started, a simple Docker_File task can be tested by using this Dockerfile as the content of the _Script_ section (task implementation)
+
+----
+FROM ubuntu:18.04
+RUN echo "Hello ProActive" 
+RUN sleep 30
+----
+
+It will create an image using the Docker build command, by specifying commands to start (echo and sleep) for every running containers from this image. At the end the built image and the started container are deleted.
+
+The build, start, stop and remove commands can be parametrized through optional command line options explained after.
+
+* The Docker_File task allows to set parameters to the `docker build` command with regard to the docker-build CLI https://docs.docker.com/engine/reference/builder/[reference].
+----
+docker build [OPTIONS] PATH | URL | -
+----
+
+To define a _docker-build option_, use the generic information *docker-build-options* 
+
+For instance by using the docker-build-options generic information with the value --add-host, a custom host-to-IP mapping will be added to the image.
+
+* The Docker_File task allows to set parameters to the `docker run` command with regard to the docker-run CLI https://docs.docker.com/engine/reference/run/[reference].
+----
+docker run [OPTIONS] IMAGE[:TAG|@DIGEST] [COMMAND] [ARG...]
+----
+
+To define a _docker-run option_, use the generic information *docker-run-options* 
+
+For instance by using the docker-run-options generic information with the value -d=true, the container will be started in detached mode.
+
+* The Docker_File task allows to set parameters to the `docker stop` command with regard to the docker-stop CLI https://docs.docker.com/engine/reference/commandline/stop/[reference].
+----
+docker stop [OPTIONS] CONTAINER [CONTAINER...]
+----
+
+To define a _docker-run option_, use the generic information *docker-stop-options* 
+
+For instance by using the docker-stop-options generic information with the value --time 30, the container will be stopped after 30s.
```

Comment:
Please consider this formulation.
For instance, the use of/using the docker-stop-options generic information with the value --time 30 makes the container stopping after 30s.